examples/chapter-list.lua

```diff
@@ -1,10 +1,19 @@
 --[[
     This script implements an interractive chapter list
-
     This script was written as an example for the mpv-scroll-list api
     https://github.com/CogentRedTester/mpv-scroll-list
 ]]
 
+local settings = { 
+    key_scroll_down = "DOWN WHEEL_DOWN",
+    key_scroll_up = "UP WHEEL_UP",
+    key_open_chapter = "ENTER MBTN_LEFT",
+    key_close_browser = "ESC MBTN_RIGHT",
+  }
+
+local opts = require("mp.options")
+opts.read_options(settings, "chapter_list", function(list) update_opts(list) end)
```

Comment:
What is the update_opts function? I don't think there's any reason to update these options after during playback.

Reply:
fixed